pkg/api/v1/apiserver/generate.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+set -e
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+
+# The following script is used to generate the solo-kit protos.
+# This script will work both in and our of the GOPATH, however, it does assume that the imported protos will be
+# available in the root level vendor folder. This script will be run as part of `make generated-code` so there
+# should be no need to run it otherwise. `make generated-code` will also vendor the necessary protos.
+ROOT=$(dirname "${BASH_SOURCE[0]}")/../../../../..
+SOLO_KIT=${ROOT}/solo-kit
+IN=${SOLO_KIT}/pkg/api/v1/apiserver
+VENDOR_ROOT=vendor_any/github.com
+
+TEMP_DIR=$(mktemp -d)
+cleanup() {
+    echo ">> Removing ${TEMP_DIR}"
+    rm -rf ${TEMP_DIR}
+}
+trap "cleanup" EXIT SIGINT
+
+echo ">> Temporary output directory ${TEMP_DIR}"
+
+IMPORTS="\
+    -I=${IN} \
+    -I=${ROOT} \
+    -I=${VENDOR_ROOT}/solo-io/protoc-gen-ext"
+
+GOGO_FLAG="--go_out=plugins=grpc:${TEMP_DIR}"
```

Comment:
nit: GO_FLAG

Reply:
also, what is this file?

Reply:
Generates the proto for the apiserver client. It's one of the solo-kit backends, I believe it's unused.